src/firmware/tools/BUILD

```diff
@@ -0,0 +1,9 @@
+sh_binary(
+    name = "flash_firmware",
```

Comment:
So we `bazel run` this target to flash old firmware?

Reply:
yep! similar to how we flash/debug the new firmware.